pyresample/spherical.py

```diff
@@ -62,35 +161,48 @@ def cross2cart(self, point):
         g = np.cos(lat1)
         h = np.cos(lat2)
         i = np.sin(lon2 - lon1)
-        res = CCoordinate(np.array([-ad * c + be * f,
-                                    ad * f + be * c,
-                                    g * h * i]))
-
+        x = -ad * c + be * f
+        y = ad * f + be * c
+        z = g * h * i
+        vertices = _xyz_to_vertices(x, y, z)
```

Comment:
Is `_xyz_to_vertices` doing the same number of copies and producing the same result as the `np.array` call was in the old version of the code?

Reply:
In the old version of the code, all the elements were only scalar values.
In `_xyz_to_vertices` we do `vertices = np.array([x, y, z])` so nothing has changed. 
   
With this PR, now we allow to work also with lon/lat arrays, so `_xyz_to_vertices` does `np.vstack([x, y, z]).T` and this creates a copy I guess, but I don't see how to avoid that.

Reply:
So I think `np.array` will do a copy no matter what you do because it is converting from a list to a numpy array. I think the same is true for `np.vstack`, but I suppose if `.T` isn't a copy (it shouldn't be...I think) then maybe this is all fine.